easybuild/easyconfigs/i/intel/intel-2016.00.eb

```diff
@@ -0,0 +1,22 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'intel'
+version = '2016.00'
```

Comment:
hmm, not sure if this is the best version, maybe we should stick to `2016.1.150`?

that is: the first update of the 2016 version of these tools; the `150` part comes back in the `impi` and `imkl` versions to indicate that all of these 'match'